scout/load/hpo.py

```diff
@@ -189,7 +182,7 @@ def load_disease_terms(
 
     if not hpo_disease_lines:
         hpo_disease_lines = fetch_hpo_to_genes_to_disease()
-    hpo_term_to_symbol = _get_hpo_term_to_symbol(hpo_disease_lines)
+    hpo_term_to_symbol = parse_hpo_to_genes(lines=hpo_disease_lines)
```

Comment:
Perhaps we should rename the argument? This "hpo_disease_lines" is kind of confusing when it will anyway be fed the "hpo_gene_lines"?!